### PR TITLE
Updated vg image to v1.4.0-1649-g8a28f7d

### PR DESCRIPTION
**Pull request by cmarkello**

### CODE REVIEW

**Comment by cmarkello** [comment]
@hannes-ucsc Ready for review.

File: vg/build/Dockerfile [hunk 1]
```diff
@@ -20,6 +24,7 @@ WORKDIR /home
 RUN git clone --recursive https://github.com/vgteam/vg.git
 
 WORKDIR /home/vg
-RUN git checkout 9aa20df8f71ebf67b0050e3c77db5f6e4cab256d
```
Comment (fnothaft) [inline vg/build/Dockerfile:23]: You should still be checking out a tag or commit from git, no?

**Review by fnothaft — COMMENTED** [review]
Did you intend to drop the checkout of a specific commit?

File: vg/build/Dockerfile (hunk 1 above)
Comment (hannes-ucsc) [reply vg/build/Dockerfile:23]: Yes, you need to pin the commit you are checking out. Especially, if you are tagging the image as a specific version, i.e. v1.4.0-1649-g8a28f7d.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
Same concern as @fnothaft

**Comment by hannes-ucsc** [comment]
Build hung for two days.

Jenkins, test this please.

**Comment by cmarkello** [comment]
@hannes-ucsc Ready for review.

**Comment by cmarkello** [comment]
@hannes-ucsc @fnothaft Ready for review.

**Review by hannes-ucsc — APPROVED** [review]

**Pull request merged**